package.json

```diff
@@ -16,20 +15,38 @@
   },
   "xo": {
     "parser": "babel-eslint",
-    "extends": ["prettier", "prettier/react", "plugin:react/recommended"],
-    "env": ["browser", "node"],
+    "extends": [
+      "prettier",
+      "prettier/react",
+      "plugin:react/recommended"
+    ],
+    "env": [
+      "browser",
+      "node"
+    ],
     "rules": {
       "linebreak-style": 0,
       "react/display-name": 0,
       "react/prop-types": 0
     },
     "space:": 2,
-    "ignores": ["next.config.js"]
+    "ignores": [
+      "next.config.js"
+    ],
+    "overrides": [
+      {
+        "files": "__tests__/*.test.js",
+        "globals": [
+          "describe",
+          "it",
+          "expect"
+        ]
+      }
+    ]
   },
   "lint-staged": {
     "*.js": [
       "prettier --write --single-quote --print-width=80 --trailing-comma=all",
-      "xo",
       "git add"
```

Comment:
It would make sense to run `xo` and `jest` only for `*.js` file changes so can you move back `xo` and new  `jest` to `lint-staged` options?

Reply:
If we add jest in lint-staged, the tests aren't being run.
To run the tests we are running the tests on precommit ([see here](https://github.com/tdeschryver/coderplex/blob/9effcff942eb3d30d97b108e2e56aad4043ae4b5/package.json#L9)).

Reply:
Yes I saw that, but why lint-staged can't run jest any reason?
I found this https://github.com/facebook/jest/pull/1678 so according to it we can pass `jest --findRelatedTests` to lint-staged to run tests for staged js files I think.